index.html

```diff
@@ -4,41 +4,61 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="style.css"/>
+    <title>About myself</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
     <header>
-        <a class="logo" href="#">My Logo</a>
-        
-            <nav class="menu_icon"><img src="images/Union.png"/></nav>
+        <section>
+            <div class="headline">
+                <main>
+                    <h1>
+                        About Myself
+                    </h1>
+                    
+                    <p>
+                        Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
```

Comment:
- [ ] Only your about section is displayed now , please make the home page and the other section like work section visible ,  this milestone is to finish the  mobile version

- Your design 👇 

![image](https://github.com/MayPyone/My_Portfolio/assets/75126481/49806160-e7e6-456c-89a2-3fc5a148a378)